docs/guide/localization/static.md

```diff
@@ -239,17 +239,91 @@ As a reminder, namespacing is primarily necessary for *installable modules* and
 
 Of course, this can be a lot of work and would likely involve tracking down where strings are used for testing purposes. If you are interested in being part of translating the UI for a language that isn't supported yet, please contact us in [Discord](http://chat.apostrophecms.com) or at [help@apostrophecms.com](mailto:help@apostrophecms.com) so we can coordinate efforts and let the whole community benefit.
 
-### Localizing your own custom Apostrophe user interface modules
+### Localizing schema field labels
+
+An important part of the admin UI that often needs localization is the field labels in your schemas. These labels appear throughout the admin interface and should be presented in the editor's preferred language. You can localize these labels using namespaced keys in your schema fields:
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  fields: {
+    add: {
+      location: {
+        type: 'string',
+        label: 'venueStrings:location'
+      },
+      capacity: {
+        type: 'integer',
+        label: 'venueStrings:capacity'
+      }
+    },
+    group: {
+      basics: {
+        label: 'venueStrings:basicInfo',
+        fields: [ 'location', 'capacity' ]
+      }
+    }
+  },
+  // Configure the namespace for browser use since these labels 
+  // appear in the admin UI
+  i18n: {
+    venueStrings: {
+      browser: true  // Makes translations available in the admin UI
+    }
+  }
+};
+```
+<template v-slot:caption>
+/modules/venue/index.js
+</template>
+</AposCodeBlock>
+
+Then add your translations to the corresponding namespace directory:
+
+<AposCodeBlock>
+
+```json
+{
+  "location": "Venue Location",
+  "capacity": "Maximum Capacity",
+  "basicInfo": "Basic Information"
+}
+```
+<template v-slot:caption>
+/modules/venue/i18n/venueStrings/en.json
+</template>
+</AposCodeBlock>
+
+<AposCodeBlock>
+
+```json
+{
+  "location": "Ubicación del lugar",
+  "capacity": "Capacidad máxima",
+  "basicInfo": "Información básica"
+}
+```
+<template v-slot:caption>
+/modules/venue/i18n/venueStrings/es.json
+</template>
+</AposCodeBlock>
+
+Since these labels appear in the admin UI, they need to be configured in your module's top-level i18n property. This property is separate from your schema configuration and contains options for each namespace used in your module. Currently, the only available option is `browser: true`, which tells Apostrophe to include these translations in the browser-side admin UI bundle. Without this setting, the translations would only be available server-side, which works for public-facing content (like Nunjucks templates) but not for the admin UI where the labels need to be available in the browser. Each namespace that needs to be available in the admin UI must be configured with `browser: true`.
```

Comment:
First sentence could explain the need better. Maybe: "Since these labels will appear in the admin UI, we need to tell Apostrophe to make sure they are available to the browser, not just on the server side." And then a fresh sentence to introduce *how* we do that.